server/container_create.go

```diff
@@ -610,6 +610,14 @@ func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerReq
 	if err := s.createContainerPlatform(container, sb.InfraContainer(), sb.CgroupParent()); err != nil {
 		return nil, err
 	}
+	defer func() {
+		if retErr != nil {
+			log.Infof(ctx, "createCtr: removing container ID %s from runtime", containerID)
+			if err2 := s.Runtime().DeleteContainer(container); err2 != nil {
+				log.Warnf(ctx, "failed to delete container in runtime %s: %v", containerID, err)
```

Comment:
I'd pref, the below, YMMV:
```suggestion
				log.Warnf(ctx, "failed to delete container %s in runtime: %v", containerID, err)
```